config/core/deployments/controller.yaml

```diff
@@ -78,5 +78,4 @@ spec:
         secret:
           secretName: google-cloud-key
           optional: true
-      serviceAccount: controller
```

Comment:
We already have `serviceAccountName` specified. `serviceAccount` is deprecated in favor of `serviceAccountName`